arcee/config.py

```diff
@@ -32,16 +32,11 @@ def get_conditional_configuration_variable(key, default):
     else:
         config = {}
 
-    if os.getenv(key) != None:
-        return os.getenv(key)
-    elif key in config.keys():
-        return config[key]
-    else:
-        return default
+    return os.environ.get(key, config.get(key, default))
```

Comment:
This will technically be identical logic to the original code because it'll handle `None`s correctly
```suggestion
    return (os.getenv(blah) or config.get(blah)) or default
```